contents/blog/vc-or-bootstrap.md

```diff
@@ -0,0 +1,101 @@
+
+We have to prepare for the board meetings - now that we've got the process down, these usually take a few hours to do. We keep the majority of the meeting focused on one or two discussion topics rather than presenting an update. The guidance our board have given us has outweighed the cost of setting up and running these sessions every 3 months. And, frankly, if I know someone is marking my homework, it encourages me to do a better job. Sidenote: is this the real reason we're open source?!
+
+## Control
+
+Our board could pressure us to take certain decisions, but Tim (cofounder) and I are ultimately in control - after our series B, we've got control of three out of five seats. Control is something we can negotiate each time we fundraise.
+
+The fact we have plenty of runway, can pay everyone well, and have great people that ship stuff, means we have _more_ control, than a bootstrapped startup that _must_ generate revenue or go out of business.
+
+Of course, this can easily result in a lack of discipline. However, it has helped us be able to get an open source project off the ground first, before focussing on revenue last summer.
+
+## Is fundraising a drag?
+
+There is a huge amount of variation. It all comes down to roughly these requirements:
+
+* clear explanation of what you're working on
+* fast progress being made
+* an idea that could be very big
+* proven team
+* good vc market conditions
+
+I've intentionally not listed "a good network" as, whilst easier for some than others, I believe you can get this done as a byproduct of the above.
+
+None of these are necessary, but the better you do against each of them, the less time you'll have to spend fundraising. Ironically, this means you'll have more time to focus on product, which means - the easier time you'll have fundraising.
+
+## More bosses
+
+Although you may retain control, if you fundraise, you've now got investors that you feel you _should_ report to.
+
+However, if you're building a bootstrapped company, you _still_ have to report to someone - your own team, if applicable, and your customers.
+
+One more party has made a positive difference to us, but for others it can be bad - if your board asks for X, and users want Y, then you may have problems. This depends on the kind of people you raise from and your relationship with them.
+
+There is a critical boss that VC removes though - you, to a certain extent. Personal finances can survive much longer when you have VC funding and when you're early stage. This can lead to a lack of discipline, as mentioned before, or it can lead to longer term thinking. It certainly feels less stressful. The fact you still need to be accountable for spending to your investors, we've found, does mean that when we spend a lot, it is intentional and reasoned.
```

Comment:
```suggestion
There is a critical boss that VC removes though – **you**, to a certain extent. Personal finances can survive much longer when you're early-stage but have  VC funding.

This can lead to a lack of discipline, as mentioned before, or it can lead to longer term thinking. But it's quantifiably less stressful. 

The fact you still need to be accountable for spending to your investors, we've found, means larger spending is intentional and reasoned.
```